flink-end-to-end-tests/flink-end-to-end-tests-elasticsearch6/src/main/java/com/apache/flink/streaming/tests/TupleC2.java

```diff
@@ -0,0 +1,30 @@
+package com.apache.flink.streaming.tests;
+
+import org.apache.flink.api.java.tuple.Tuple2;
+
+/** A {@link Comparable} implementation of {@link Tuple2}. */
+public class TupleC2<T0 extends Comparable<? super T0>, T1 extends Comparable<? super T1>>
```

Comment:
I would much rather have a dedicated class for the test results than introduce a new `Tuple` class.